tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/how_to_connect_to_in_memory_data_using_spark.py

```diff
@@ -7,6 +7,7 @@
 
 import pathlib
 import great_expectations as gx
+import pyspark.pandas as ps
```

Comment:
```suggestion
import os
import pyspark.pandas as ps

# Required by pyarrow>=2.0.0 within Spark to suppress UserWarning
os.envrion["PYARROW_IGNORE_TIMEZONE"] = "1"
```

Reply:
Is this going to be part of the snippet? I don't think it should be.

Reply:
outside the snippet